sass/themes/frost/components/_modal.scss

```diff
@@ -0,0 +1,37 @@
+.modal__overlay {
+    position: fixed; /* Stay in place */
+    z-index: 1; /* Sit on top */
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
```

Comment:
@lebaz20 I think we can use variables more here for colours and perhaps sizes - definitely when the colour is an exact match for something in `variables/_colours.scss` - and perhaps we should consider matching more of the existing vars when we're hard-coding something very similar here?